kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/tools/jacoco/PrintCoverage.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.gradle.plugin.tools.jacoco
+
+import kotlinx.kover.features.jvm.CoverageValue
+import kotlinx.kover.features.jvm.RuleViolations
+import kotlinx.kover.gradle.plugin.commons.KoverCriticalException
+import kotlinx.kover.gradle.plugin.commons.ReportContext
+import kotlinx.kover.gradle.plugin.commons.VerificationBound
+import kotlinx.kover.gradle.plugin.commons.VerificationRule
+import kotlinx.kover.gradle.plugin.tools.CoverageRequest
+import kotlinx.kover.gradle.plugin.tools.writeNoSources
+import kotlinx.kover.gradle.plugin.tools.writeToFile
+import kotlinx.kover.gradle.plugin.util.ONE_HUNDRED
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.Property
+import org.gradle.workers.WorkQueue
+import java.io.File
+import java.math.BigDecimal
+
+internal fun ReportContext.printJacocoCoverage(request: CoverageRequest, outputFile: File) {
+    val bound = VerificationBound(ONE_HUNDRED, BigDecimal.ZERO, request.metric, request.aggregation)
+    val failRule = VerificationRule(true, "", request.entity, listOf(bound))
```

Comment:
Hm, is this the rule that we apply by default if none are specified?

Reply:
No, this is a rule that will never passed (min=100, max=0).
Thus, JaCoCo will inform us of the current coverage value in error message.

Since JaCoCo doesn't just have an API for explicitly obtaining coverage values, we get them indirectly through verification.

Reply:
> Since JaCoCo doesn't just have an API for explicitly obtaining coverage values, we get them indirectly through verification.

I think this info is worth adding as a comment